lib/mappers/request.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
```

Comment:
Are these "global" mappings (i.e. mappings that apply to more than one specific method?).

Can you add a heading comments explaining what are these for (how they are different from the method-specific mappings).

Also the word "request" is kind of generic and is used with other meaning elsewhere in the codebase, so we should come up with a differnt one.